### PR TITLE
Bump storybook and add breakpoint viewports

### PR DESCRIPTION
**Pull request by kyledurand**
We lost the viewport toggle at some point, looks like it moved to an addon so I've added it back and mapped it to our breakpoints. This will help us ship updates to opinionated layout components that change margins / paddings / orientation etc...at our different breakpoints

image

https://user-images.githubusercontent.com/6844391/196984820-4ddbbed1-00e4-4ab0-a1cb-d5a96f049242.mp4

### CODE REVIEW

**Review by aveline — APPROVED** [review]
🎉 Thanks for doing this. Great that we can pull in the breakpoints from our tokens!

File: polaris-react/.storybook/preview.js [hunk 1]
```diff
@@ -79,10 +80,27 @@ export const globalTypes = {
   },
   ...gridOptions,
 };
+const views = {...breakpoints};
+delete views['breakpoints-xs'];
```
Comment (aaronccasanova) [inline polaris-react/.storybook/preview.js]: Light suggestion to override this breakpoint value with something like `320px` (with a brief comment explaining why). That way we can still test the `xs` breakpoint (e.g. 0-489px viewports).

Comment (aaronccasanova) [reply polaris-react/.storybook/preview.js]: I believe we do the same in playroom cc @gwyneplaine

Comment (kyledurand) [reply polaris-react/.storybook/preview.js]: I like that 👍

**Review by laurkim — APPROVED** [review]
Tested locally and looks great, nice! :shipit: 💯

**Review by alex-page — APPROVED** [review]

**Pull request merged**